### DISCUSSION TITLE
What is the difference between `auth-mailchimp-sync` and the official mailchimp extension?

### DISCUSSION THREAD

**Discussion by smac89** [firebase/extensions#862] [Q&A] [answered]
I'm a slightly conflicted when it comes to deciding between  `auth-mailchimp-sync` and https://firebase.google.com/products/extensions/mailchimp-mailchimp-firebase-sync.
Can someone explain the difference between these two. I just want to be able to add new users from firebase to mailchimp.

**Comment by dackers86** [comment, marked as answer]
Hi @smac89 

The Mailchimp extension on this repository was the original version and is now deprecated. 

The official (and recommended) version is now hosted by Mailchimp, so https://firebase.google.com/products/extensions/mailchimp-mailchimp-firebase-sync  is the best option going forward.

**Reply by smac89** [reply]
> Hi @smac89 
> 
> The Mailchimp extension on this repository was the original version and is now deprecated. 
> 
> The official (and recommended) version is now hosted by Mailchimp, so https://firebase.google.com/products/extensions/mailchimp-mailchimp-firebase-sync  is the best option going forward.

Thanks for the reply
